templates/Template k8s-galera.xml

```diff
@@ -0,0 +1,187 @@
+        <template>
+            <uuid>dfb8263cf05842f5bc3d8d7f3f11bcbf</uuid>
+            <template>Template k8s-galera mariadb status</template>
+            <name>Template k8s-galera mariadb status</name>
+            <groups>
+                <group>
+                    <name>Templates/Applications</name>
+                </group>
+            </groups>
+            <discovery_rules>
+                <discovery_rule>
+                    <uuid>b1134c1839a5481eb653131795ad65ce</uuid>
+                    <name>Galera cluster discovery</name>
+                    <type>ZABBIX_ACTIVE</type>
+                    <key>k8s-galera.discovery-clusters</key>
+                    <delay>1h</delay>
+                    <description>discovery of the mariadb clusters</description>
+                    <item_prototypes>
+                        <item_prototype>
+                            <uuid>0c03b14cfb904f8aaef6a74b493b3598</uuid>
+                            <name>galera cluster status {#CLUSTER} current_primary_pod_index</name>
+                            <type>DEPENDENT</type>
+                            <key>k8s-galera.current_primary_pod_index[{#CONTEXT},{#NAMESPACE},{#CLUSTER}]</key>
+                            <delay>0</delay>
+                            <history>1d</history>
+                            <value_type>TEXT</value_type>
+                            <trends>0</trends>
+                            <preprocessing>
+                                <step>
+                                    <type>JSONPATH</type>
+                                    <parameters>
+                                        <parameter>$.current_primary_pod_index</parameter>
+                                    </parameters>
+                                </step>
+                            </preprocessing>
+                            <master_item>
+                                <key>k8s-galera.values-clusters[{#CONTEXT},{#NAMESPACE},{#CLUSTER}]</key>
+                            </master_item>
+                            <trigger_prototypes>
+                                <trigger_prototype>
+                                    <uuid>3dc862e0a0ee4cd7ac9160a97e55fd34</uuid>
+                                    <expression>change(/Galera/k8s-galera.current_primary_pod_index[{#CONTEXT},{#NAMESPACE},{#CLUSTER}])=1</expression>
+                                    <name>Galera {#CONTEXT}:{#CLUSTER} primary pod changed</name>
+                                    <priority>DISASTER</priority>
+                                    <description>The primary Galera pod changed</description>
+                                </trigger_prototype>
+                            </trigger_prototypes>
+                        </item_prototype>
+                        <item_prototype>
+                            <uuid>c529c403c1c347da87deadca49fe0c5a</uuid>
+                            <name>galera cluster status {#CLUSTER} current_replica_count</name>
```

Comment:
if i remember correctly the name need to be unique per host. what if a customer had 2 clusters both named galera in there kubenetes cluster, one in production namespace and the other in acceptance. if you dont have the namespace name in the item name this would not work

Reply:
maybe use {#CONTEXT}:{#NAMESPACE}:{#CLUSTER} everywhere?

Reply:
I'm now using `{#CONTEXT}:{#NAMESPACE}:{#CLUSTER}` everywhere